mlflow/openai/utils.py

```diff
@@ -99,3 +104,75 @@ def request(*args, **kwargs):
             return original(*args, **kwargs)
 
     return _mock_request(new=request)
+
+
+def _get_task_model_params(model: Dict[str, Any], params: Dict[str, Any] = None):
+    model_dict = model.copy()
+    model_params = params.copy()
```

Comment:
@serena-ruan  do you mean the `copy()` or the method itself? The reason for `copy()` is because we use `pop`. We alter the dictionary. But it's not required for params. Let me know if it make sense to you.